### PR TITLE
GuildGamble Gold Cap

### PR DESCRIPTION
**Pull request by JukeDragh**

### CODE REVIEW

**Comment by JukeDragh** [comment]
I also don't know how to deploy everything

File: src/plugins/events/events/GuildGambling.js [hunk 1]
```diff
@@ -21,9 +21,10 @@ export class GuildGambling extends Event {
     if(tavernLevel === 0) return;
 
     const MOD_INCOME_PERCENT = INCOME_PERCENT + Math.floor(tavernLevel / 5);
+    const MAX_GOLD = math.min(player.gold / MOD_INCOME_PERCENT, tavernlevel * 200000);
```
Comment (Kookiecraft) [inline src/plugins/events/events/GuildGambling.js]: This should be Math.min

Comment (JukeDragh) [inline src/plugins/events/events/GuildGambling.js]: Thx

**Comment by seiyria** [comment]
@Kookiecraft if this sounds good to you, feel free to merge it. It looks good to me.

**Pull request merged**